ironfish/src/telemetry/telemetry.ts

```diff
@@ -114,9 +117,35 @@ export class Telemetry {
   private metricsLoop(): void {
     Assert.isNotNull(this.metrics)
 
+    const inboundTrafficFields: Field[] = [
+      ...this.metrics.p2p_InboundTrafficByMessage.entries(),
+    ].map(([messageType, meter]) => {
+      return {
+        name: 'inbound_traffic_' + NetworkMessageType[messageType].toLowerCase(),
+        type: 'float',
+        value: meter.rate5m,
+      }
+    })
+
+    const outboundTrafficFields: Field[] = [
+      ...this.metrics.p2p_OutboundTrafficByMessage.entries(),
+    ].map(([messageType, meter]) => {
+      return {
+        name: 'outbound_traffic_' + NetworkMessageType[messageType].toLowerCase(),
+        type: 'float',
+        value: meter.rate5m,
+      }
+    })
+
     this.submit({
       measurement: 'node_stats',
       timestamp: new Date(),
+      tags: [
+        {
+          name: 'synced',
+          value: this.chain.synced.toString(),
+        },
+      ],
```

Comment:
this will partition all the fields into 2 tables, I think this should be a measurement right?